Model/Vote.php

```diff
@@ -95,7 +96,14 @@ public function isVoteValid(ExecutionContextInterface $context)
         if (!$this->checkValue($this->value)) {
             $message = 'A vote cannot have a 0 value';
             $propertyPath = $context->getPropertyPath() . '.value';
-            $context->addViolationAt($propertyPath, $message);
+
+            if (version_compare(Kernel::VERSION, '2.5.0', '<')) {
```

Comment:
no, because it always exist. The check is easy though thanks to the new ExecutionContextInterface being added:

```
if ($context instanceof \Symfony\Component\Validator\Context\ExecutionContextInterface) {
    // Validator 2.5 API is available
} else {
    // legacy mode
}
```